netbout-web/src/main/js/bout.js

```diff
@@ -69,31 +69,51 @@ $(document).ready(
               dataType: 'xml',
               method: 'GET',
               success: function (data) {
-                var appendix = '';
+                var appendix = '', $data = $(data), messagesAsXML = $data.find('message'),
```

Comment:
@erimerturk Let's avoid declaring multiple variables on the same line as it makes the code hard to read.